### PR TITLE
TodoList.js and AddTodoForm.js added

### PR DESCRIPTION
**Pull request by yasi44**
TodoList.js and AddTodoForm.js added

### CODE REVIEW

File: src/TodoList.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import react from "react";
```
Comment (sinancengiz) [inline src/TodoList.js]: we might want React as first letter capital

File: src/TodoList.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      id:2,
+      title:"Make an appointment"
+    },
+    {
+      id:3,
+      title:"Call mom"
+    }
+  ]; 
+
+let TodoList = () => {
```
Comment (sinancengiz) [inline src/TodoList.js]: we want to declare component function as const instead of let

File: src/AddTodoForm.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+let AddTodoForm = () => {
```
Comment (sinancengiz) [inline src/AddTodoForm.js]: same here const instead of let

**Review by sinancengiz — APPROVED** [review]
Great work, just couple minor suggestion and LGTM

**Review by yasi44 — COMMENTED** [review]
reviewed

**Pull request merged**